app/models/application_record.rb

```diff
@@ -1,3 +1,5 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+  mount_uploader :avatar, AvatarUploader
+  validates :username, :firstname, :lastname, :email, :bio, :presence => true
```

Comment:
Here's the guide from GitHub: [About pull requests
](https://help.github.com/articles/about-pull-requests/)